perses/rjmc/topology_proposal.py

```diff
@@ -338,35 +341,55 @@ def _get_mol_atom_map(current_oemol,
              else: raise Exception.
              NOTE : the matching criterion pulls patterns and target matches based on indices or names;
                     if 'names' is chosen, it is first asserted that the current_oemol and the proposed_oemol have atoms that are uniquely named
-
+        external_inttypes : bool, default False
+            If True, IntTypes already assigned to oemols will be used for mapping, if IntType is in the atom or bond expression.
+            Otherwise, IntTypes will be overwritten such as to ensure rings of different sizes are not matched.
+        return_all_maps : bool, default False
+            will return a list of viable maps. This is for debugging purposes only
+            as a list being returned in the overall pipeline would break everything
+        geometry : str, default 'weak'
+            one of ['strong','weak' or None]
+            if strong -- atoms that are close in the geometries of ligand A and B are forced to map together
+            if weak -- if there are multiple mappings, the mapping that is closer to the geometry of ligand B is chosen
+            if None, all geometry information is ignored
         Returns
         -------
         matches : list of match
```

Comment:
the object that is returned is now inconsistent. perhaps if geometry is used and returns the best match, it should simply return a list of length 1?

Reply:
Possibly --- but that is still going to potentially break other things further down the line. I've added _logger.warning that returning all matches shouldn't be done in the normal pipeline